bkrdoc/analysis/linter/output_generator.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+__author__ = 'blurry'
+
+from bkrdoc.analysis.parser import bkrdoc_parser
+from bkrdoc.analysis.linter import linter
+
+class OutputGenerator(object):
+    file_ = ""
+    parser = ""
+    # phases = []
+
+    def __init__(self, file):
+        self.parser = bkrdoc_parser.Parser(file)
+        self.parser.parse_data()
+
+    def analyse(self):
+
+        #for elem in self.parser.argparse_data_list:
+        #    print(elem)
+
+        _linter = linter.Linter()
```

Comment:
kind of :D yes, this is bad
i was aware of "private" python variables, but this is me first time hearing of protected ones, i was used to prefixing with underscore in other languages, mea culpa